README.md

```diff
@@ -133,6 +149,24 @@ f = f2 >>= f1
 
 </details>
 
+<details><summary>Function as Semigroup</summary>
+
+```haskell
+f x = f1 x <> f2 x
+f = f1 <> f2
```

Comment:
I would suggest add a few more to be more explicit on compose-ability / associativity.

```haskell
f x = f1 x <> f2 x <> f3 x <> fn x
f = f1 <> f2 <> f3 <> fn
```